jenkins_repo

```diff
@@ -0,0 +1 @@
+hi trying to fork the repo for practise
```

Comment:
Please don't waste the time of open source contributors and the resources of open source projects by submitting junk pull requests.  Practice on your own repositories, not on repositories maintained by others.

Reply:
Doesn't even need to be one's own. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests tells readers to work with https://github.com/octocat/Spoon-Knife